### PR TITLE
created config

### PR DESCRIPTION
**Pull request by chris-soelistyo**
- Created config file, and run file to go through the whole training pipeline.

### CODE REVIEW

File: grace/run.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+from grace.models.datasets import dataset_from_graph
+from grace.models.classifier import GCN
+from grace.models.feature_extractor import FeatureExtractor
+from grace.utils.transforms import get_transforms
+
+
+@click.command(name="GRACE Trainer")
+@click.option("--config_file", type=click.Path(exists=True))
+def run(config_file: Union[str, os.PathLike]) -> None:
+    """Runs the GRACE pipeline; going straight from images and annotations
```
Comment (mooniean) [inline grace/run.py:22]: Slight change: going from images and .grace annotations, so it's not confused with annotations in starfiles or other types

File: grace/io/image_dataset.py [hunk 2]
```diff
@@ -28,20 +28,22 @@ class ImageGraphDataset(Dataset):
...
         *,
         transform: Callable = lambda x, g: (x, g),
+        image_filetype: str = "mrc",
     ) -> None:
-        self.image_reader_fn = image_reader_fn
+        self.image_reader_fn = FILETYPES[image_filetype]
         self.transform = transform
 
-        image_paths = list(Path(image_dir).iterdir())
+        image_paths = list(Path(image_dir).glob(f"*.{image_filetype}"))
```
Comment (mooniean) [inline grace/io/image_dataset.py:46]: what if there's no images of that filetype?

Comment (chris-soelistyo) [reply grace/io/image_dataset.py:46]: @mooniean The code fails then. The user should input the filetype expected.

File: grace/run.py (hunk 1 above)
Comment (chris-soelistyo) [reply grace/run.py:22]: @mooniean will do!

**Review by mooniean — APPROVED** [review]
LGTM!

**Pull request merged**